test/common/stats/histogram_impl_test.cc

```diff
@@ -0,0 +1,61 @@
+#include "test/mocks/stats/mocks.h"
+
+#include "gtest/gtest.h"
+
+using testing::_;
+
+namespace Envoy {
+namespace Stats {
+namespace {
+
+TEST(HistogramImplTest, Basic) {
+  MockStore store;
```

Comment:
Can you make a test-class that captures these 4 lines of declaration? Then (most) every stanza can be a test method.

Reply:
Each "stanza" builds on the previous, they don't really stand alone. The only one that could be its own test is the initial portion about names. But I could buy the "one assert per test" argument too... Do you have strong feelings about this?